<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmResoType</class>
 <widget class="QDialog" name="frmResoType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>50</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>选择谐振点类型</string>
  </property>
  <property name="styleSheet">
   <string notr="true">frmResoType
{
	
	background-color: rgb(144, 148, 248);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btnSeries">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>串联</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnParallel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>并联</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCrystal">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>晶体谐振器</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setResonaceType()</slot>
 </slots>
</ui>
